src/Dscp/Crypto/Encrypt.hs

```diff
@@ -22,16 +21,21 @@ module Dscp.Crypto.Encrypt
        ) where
 
 import Crypto.Cipher.AES (AES256)
-import Crypto.Cipher.Types (AEAD, AEADMode (AEAD_GCM), AuthTag, BlockCipher (aeadInit),
+import Crypto.Cipher.Types (AEAD, AEADMode (AEAD_GCM), AuthTag (..), BlockCipher (aeadInit),
                             Cipher (cipherInit), IV, aeadSimpleDecrypt, aeadSimpleEncrypt, nullIV)
 import Crypto.Error (onCryptoFailure)
 import Data.ByteArray (ByteArray, ByteArrayAccess)
 import qualified Data.ByteArray as BA
+import qualified Data.ByteString as BS
 import Data.Text.Buildable (build)
 import Fmt ((+|), (|+))
+import Test.QuickCheck (Arbitrary (..), suchThatMap, vector)
+import Test.QuickCheck.Instances ()
```

Comment:
No, please, let's keep all `Arbitrary` instances in `tests` (or at least in separate `*.Arbitrary` modules).
I'd really prefer to keep them only in tests unless really necessary otherwise (like in case with using these instances in benchmarks).

Reply:
Yep, will do